luarules/gadgets/unit_air_attacksafetydistance.lua

```diff
@@ -13,20 +13,22 @@ function gadget:GetInfo()
 end
 
 --[[
-	Wiki: "attackSafetyDistance" movetypedata: Fighters abort dive toward target if within attackSafetyDistance and
+	springrts Wiki: "attackSafetyDistance" movetypedata: Fighters abort dive toward target if within attackSafetyDistance and
 	try to climb back to normal altitude while still moving toward target.	It's disabled by default. Set to half of
 	the minimum weapon range to avoid collisions, enemy fire, AOE damage. If set to greater than the weapon range,
 	the unit will fly over the target like a bomber.
 ]]--
 
 if not gadgetHandler:IsSyncedCode() then
-	return
+	return false
 end
 
-local isFighter = {}
+local attackSafetyDistance = {}
 for udid, ud in pairs(UnitDefs) do
-	if ud.canFly and ud.customParams.fighter then
-		isFighter[udid] = true
+	if ud.canFly and ud.isStrafingAirUnit then 
+		if ud.customParams.attacksafetydistance then
```

Comment:
these two could be a single condition with an `and` rather than two `if`s, but it's a nitpick and I already approved the PR